src/main/resources/assets/refinedrelocation/lang/de_DE.lang

```diff
@@ -0,0 +1,109 @@
+tile.refinedrelocation:sorting_chest.name=Sortierende Truhe
+tile.refinedrelocation:block_extender.name=Blockverlängerung
+tile.refinedrelocation:fast_hopper.name=Schneller Trichter
+tile.refinedrelocation:filtered_hopper.name=Gefilterter Trichter
+tile.refinedrelocation:buffer.name=Puffer
+tile.refinedrelocation:sorting_connector.name=Sortierverbindung
+
+item.refinedrelocation:sorting_upgrade.name=Sortierupgrade
+item.refinedrelocation:stack_limiter.name=Stapelbegrenzung
+item.refinedrelocation:input_filter.name=Inputfilter
+item.refinedrelocation:output_filter.name=Outputfilter
+
+container.refinedrelocation:root_filter=Filter bearbeiten
+container.refinedrelocation:root_filter_with_name=%s (Filter)
+container.refinedrelocation:sorting_chest=Sortiertruhe
+container.refinedrelocation:root_filter=Filter bearbeiten
+container.refinedrelocation:name_filter=Filter benennen
```

Comment:
This is the header for the GUI of the name filter, so it should be called the same (Namensfilter)